agent/consul/leader_peering_test.go

```diff
@@ -747,10 +747,10 @@ func TestLeader_Peering_ImportedServicesCount(t *testing.T) {
 	/// finished adding services
 
 	type testCase struct {
-		name                          string
-		description                   string
-		exportedService               structs.ExportedServicesConfigEntry
-		expectedImportedServicesCount uint64
+		name                                  string
+		description                           string
+		exportedService                       structs.ExportedServicesConfigEntry
+		expectedImportedExportedServicesCount uint64 // same count for a server that imports the services form a server that exports them
```

Comment:
We should keeping two explicit counts here in case there's ever a need to check for different numbers (or non-symmetric imports/exports) in the future.

Reply:
Thanks for offering feedback on this PR 💯 

---

Let me address this point in an upcoming PR.